### PR TITLE
Updating remaining DAGs docker_conn_id

### PR DESCRIPTION
**Pull request by Charlie-Henry**
Adding `docker_conn_id="docker_default",` to the the remaining `DockerOperator` tasks

## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/13726

## Associated repo
https://github.com/cityofaustin/atd-airflow/

## Testing

**Steps to test:**
I guess I'm not sure how to test this? Maybe someone has some crazy regex that can find if I missed any. We might want to check that these changes will not reschedule DAGs who's schedule was turned off? 

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Add note to 1PW secrets moved to API vault and check for duplicates

### CODE REVIEW

File: dags/atd_knack_signs_markings_time_logs.py [hunk 1]
```diff
@@ -85,18 +86,19 @@
     t2 = DockerOperator(
         task_id="atd_knack_markings_time_logs_to_postgrest",
         image=docker_image,
+        docker_conn_id="docker_default",
         auto_remove=True,
         command=f"./atd-knack-services/services/records_to_postgrest.py -a {app_name} -c {container_markings} {date_filter_arg}",
         environment=env_vars,
         tty=True,
-        force_pull=True,
```
Comment (johnclary) [inline dags/atd_knack_signs_markings_time_logs.py:92]: i see this is not needed because the previous task pulls the same image 👍

**Review by johnclary — APPROVED** [review]
it passes the eyeball test—and i'm wearing glasses! 🚢 🚢 🚢 🚢

**Review by chiaberry — APPROVED** [review]
👍🏼

**Review by mddilley — APPROVED** [review]
thank you Charlie!! looks great 🐳 🚢 🐳 🚢

**Comment by Charlie-Henry** [comment]
ty pals!

**Pull request merged**